modules/caddyhttp/reverseproxy/healthchecks.go

```diff
@@ -353,7 +353,7 @@ func (h *Handler) doActiveHealthCheck(dialInfo DialInfo, hostAddr string, upstre
 	// passed health check parameters, so mark as healthy
 	if upstream.setHealthy(true) {
 		h.HealthChecks.Active.logger.Info("host is up", zap.String("host", hostAddr))
-		h.event.AsyncDispatch(NewActiveHealthyEvent(hostAddr))
+		h.events.Emit(h.ctx, "healthy", map[string]interface{}{"host": hostAddr})
```

Comment:
Can we do some sugar for making the map? Seems like it would be annoying to make a map like this everywhere. I dunno.

Reply:
It looks much tidier when using `any` instead of `interface{}`, but yeah, the only sugar would be a type definition. I'd prefer to only do that if we actually need to lean on the type though. So far there's no behavior or type assertions that would require that.